.travis.yml

```diff
@@ -26,7 +26,7 @@ install:
 
       conda config --set show_channel_urls true
       conda update --yes conda
-      conda install --yes conda-build jinja2 anaconda-client
+      conda install --yes conda-build=1.20.0 jinja2 anaconda-client
```

Comment:
I know it generated this, but we should not include it. See this issue ( https://github.com/conda-forge/hdf5-feedstock/issues/23 ) for background as to why we removed it.